sqlparser/sqlparser.go

```diff
@@ -223,6 +224,27 @@ func (p *Parser) replaceInsertValueFromValArg(query *InsertQuery, colIndex int,
 				Val:  []byte(fmt.Sprintf("%d", arg)),
 			}
 		}
+	case time.Time:
+		query.ColumnValues[colIndex] = func() *vtparser.SQLVal {
+			return &vtparser.SQLVal{
+				Type: vtparser.StrVal,
+				Val:  []byte(arg.Format("2006-01-02 15:04:05")),
+			}
+		}
+	case *time.Time:
+		query.ColumnValues[colIndex] = func() *vtparser.SQLVal {
+			return &vtparser.SQLVal{
+				Type: vtparser.StrVal,
+				Val:  []byte(arg.Format("2006-01-02 15:04:05")),
+			}
+		}
+	case nil:
```

Comment:
I also concern about other pointer types like `*int` or `*uint` and so on..

Reply:
https://github.com/knocknote/octillery/pull/17/commits/4a79d668613579752f09f1ff1616b4b9b8800e90